app/services/darktide.server.ts

```diff
@@ -213,16 +213,32 @@ export async function getCharacterWallet(auth: AuthToken, characterId: string) {
 	}
 }
 
-let MissionBoardSchema = z.object({
+export let MissionBoardSchema = z.object({
```

Comment:
```suggestion
let MissionBoardSchema = z.object({
```

imo other code shouldn't be depending on this